src/components/KDropdown/KDropdownItem.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <li
     class="k-dropdown-item"
-    :class="{
+    :class="[{
       'has-divider': hasDivider,
       'disabled': disabled,
       'danger': danger || isDangerous,
-      'dropdown-selected-option': selected
-    }"
+      'dropdown-selected-option': selected,
+    }, $attrs.class]"
```

Comment:
question: should we purposefully bind inline styles to the parent as well? I don't think they should fallthrough

Reply:
I'm fine with whichever we pick

Reply:
Going on with your suggestion to bind inline styles to the parent